pyscitt/pyscitt/client.py

```diff
@@ -27,11 +29,19 @@
 class HttpSig(httpx.Auth):
     requires_request_body = True
 
-    def __init__(self, key_id: str, pem_private_key: str):
-        self.key_id = key_id
-        self.private_key = load_pem_private_key(
-            pem_private_key.encode("ascii"), password=None, backend=default_backend()
-        )
+    def __init__(self, key_id: str, pem_private_key: str, akv_configuration: str):
```

Comment:
Might be worth making the `key_id` an `Optional[str]`, see https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type